test/e2e/framework/metrics_util.go

```diff
@@ -184,14 +184,10 @@ var InterestingControllerManagerMetrics = []string{
 }
 
 var InterestingKubeletMetrics = []string{
-	"kubelet_container_manager_latency_microseconds",
-	"kubelet_docker_errors",
```

Comment:
it looks like these are now in the dockershim to be clear https://github.com/kubernetes/kubernetes/commit/331628b7dcd637f3e5622b654977955ebb17ce5a#diff-bb5d427394b7c4081ea5dba3b3465b76

Reply:
@yujuhong looks like you moved these, thoughts on this?

Reply:
Maybe replace this by `docker_operations_errors` is good one .

Reply:
/cc @brancz

Reply:
That sounds good to me :+1:

Reply:
@brancz @BenTheElder :  added the new metrics `kubelet_docker_operations_errors_total` to replace the deprecated/missing metrics `kubelet_docker_errors` . PTAL